File: Python/Código (OBI 2015).py
#link (https://neps.academy/problem/47)

n = int(input())
numeros = input()
cont = 0
for x in range(n):
    try:
        if numeros[x]=="1" and numeros[x+1]=="0" and numeros[x+2]=="0":
            cont = cont + 1
    except:
        break
print(cont)

File: C-C++/Código (OBI 2015).cpp
// Example program
#include <iostream>
#include <string>
using namespace std;
// link (https://neps.academy/problem/47)
int main()
{
    int n;
    cin >> n;
    int vetor[n];
    for (int x=0;x<n;x++){
        cin >> vetor[x];
    }
    int cont = 0;
    for (int x=0;x<n-2;x++){
        if (vetor[x]==1 and vetor[x+1]==0 and vetor[x+2]==0){
            cont++;
        }
    }
    cout << cont << endl;

}
